website/docs/reference/data-test-configs.md

```diff
@@ -275,3 +275,24 @@ tests:
 ```
 
 </File>
+
+#### Specify custom configurations for generic data tests
+
+_Currently available in dbt Cloud only. Specifying custom configurations for data tests will become available in dbt Core later this year._
+
+Use any custom config key to specify custom configurations for data tests. For example, the following specifies the `snowflake_warehouse` custom config that dbt should use when executing the `accepted_values` data test. Given the config, the data test runs on a different Snowflake virtual warehouse than the one in your default connection to enable better price-performance with a different warehouse size, or more granular cost allocation and visibility.
+
+```yml
+
+models:
+  - name: my_model
+    columns:
+      - name: color
+        tests:
+          - accepted_values:
+              values: ['blue', 'red']
+              config:
+                severity: warn
+                snowflake_warehouse: my_warehouse
+
+```
```

Comment:
...and put the follow up down here

```suggestion
```

Given the config, the data test runs on a different Snowflake virtual warehouse than the one in your default connection to enable better price-performance with a different warehouse size or more granular cost allocation and visibility.
```